rir/src/compiler/pir/instruction.h

```diff
@@ -1138,6 +1139,21 @@ class FLIE(MkArg, 2, Effects::None()) {
     bool usesPromEnv() const;
 };
 
+class FLIE(FakeMkArg, 2, Effects::None()) {
```

Comment:
I still claim that this is not a good idea. They should be normal MkArg's and I don't understand why we have these. In the worst case just compile them again...